### ISSUE TITLE
Inconsistent and confusing option defaults for a number of commands 

### ISSUE THREAD

**Issue by adswa** [datalad/datalad-gooey#179]
In the complex and simple command view, the 'get data' option of ``get`` is unticked. Thus, when parametrizing a 'get' call for directories or files and not selecting any other options, the call will have the equivalent of ``datalad get -n`` set:
```
Running: Dataset('C:\\Users\\adina\\repos\\datalad-gooey\\sf').get(get_data=False, path=[WindowsPath('C:/Users/adina/repos/datalad-gooey/sf/sub-01/ses-localizer')])
```

This is somewhat inconsistent to the command line, where one needs to set an extra option to **not** get data (i.e., the default in the command line get is to get data). Its also potentially confusing to users, who selected a command called "get" and don't see any data being retrieved after running it. Only in the case of subdatasets a get without data retrieval makes sense, in all other cases I feel like it adds confusion, thus I would suggest to either have it ticked by default, or revert its logic (i.e., 'not get data') & rename it to something that specifically conveys its meaning for subdatasets, such that one can tick it when specifically working with an uninstalled subdataset.

**Comment by mih** [comment]
I can confirm that the widget is unchecked, even when the signature of the command has `True` has the default.

**Comment by bpoldrack** [comment]
Probably will be addressed by #175.

**Comment by adswa** [comment]
Similarly, ``create``'s 'Dataset with file annex' option is unset by default (so any dataset running with the default will have a .noannex file):
![image](https://user-images.githubusercontent.com/29738718/191745260-63d55c0b-3e7e-4f7f-b684-d1130b41b0f4.png)
I suspect #175 might address this, too
